### DISCUSSION TITLE
Load existing index (Pinecone) not working when already upserted?

### DISCUSSION THREAD

**Discussion by gslrt** [FlowiseAI/Flowise#729] [Q&A]
Hi there 🙌

I'm chasing this issue for multiple hours now but can't find a solution... 🤪

ISSUE: When asking about my custom data it doesn't have any clue and only responds with "I don't know"

What I'm doing:

- Creating embeddings directly via OpenAI with the recommended **text-embedding-ada-002**
- Upserting them to Pinecone with some metadata following their upserting template logic
- Being able to fetch these via Pinecone console -> can't interpret the vectors values... is this somehow possible? (see next screenshot)
- Qadrouple-checking everything also with ChatGPT(4) to make sure I dont miss anything.
- Getting no error logs or anything....have some logs in place to check the input to OpenAi Embedding for example... all fine.

![image](https://github.com/FlowiseAI/Flowise/assets/140710359/233fea08-ebc4-4792-a7e6-ac7cbe090b24)
That is how my already upserted structure looks like

HOWEVER

When I upsert a .txt via Flowise (Pinecone Upsert Document) and then retrieving it via the **Pinecone Load existing index**  its working just perfect and it knows exactly the answer.

All credentials, index, namespace and so on are exactly the same

![image](https://github.com/FlowiseAI/Flowise/assets/140710359/248621f5-f665-4a2d-a8c7-9aee4cec04d7)
No matter if i ask about things I uploaded via Flowise (which I get a correct answer for) or directly to Pinecone (where it doenst know about it) I dont see requests made for Queries.

Do you experienced something similiar? 
Do you know which ID (for comparison) theses vectors generated via Flowise has?

Thank you sooo much for any guidance on that! ✨

**Comment by HenryHengZJ** [comment]
have you looked at the logs to see if any source documents were retrieved?

for example:
<img width="802" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/26460777/aefa106b-db20-40c3-bd62-14b158790688">
